### DISCUSSION TITLE
I can't change the language

### DISCUSSION THREAD

**Discussion by imax12** [EssentialsX/Essentials#4184] [Q&A] [answered]
Hello I can't change the language! If I drag the language file German down to the directory and enter it in the config as local language "" ! I can't do

**Comment by triagonal** [comment, marked as answer]
Make sure you've uncommented the locale config option by removing the `#` in front of the option.

Wrong (commented out):
```yaml
#locale: de
```
Correct (uncommented):
```yaml
locale: de
```

If you've already done the above, please paste your `plugins/Essentials/config.yml` file at https://paste.gg/ and provide the link here.
